db/init_db.js

```diff
@@ -1,31 +1,131 @@
-// code to build and initialize DB goes here
 const {
   client
   // other db methods 
 } = require('./index');
 
+const { 
+  createProduct 
+} = require('./products');
+
 async function buildTables() {
   try {
     client.connect();
+    
+    console.log('Dropping All Tables...');
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS users;
+      DROP TABLE IF EXISTS products;
+    `);
+    console.log('Finished dropping tables!');
 
-    // drop tables in correct order
-
-    // build tables in correct order
-
+    console.log('Starting to build tables...');
+    await client.query(`
+    CREATE TABLE products(
+      id SERIAL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL,
+      description TEXT NOT NULL,
+      price DECIMAL(10,2) NOT NULL,
+      "imageUrl" TEXT NULL,
```

Comment:
Please put in a default value.  This should be a string representation of a url of an actual image.  I.e. `'https://image.shutterstock.com/image-photo/soap-260nw-334591208.jpg'`